API.md

```diff
@@ -384,8 +387,24 @@ interface heatmapProp {
 
 #### Important Note
 
-If you have multiple `GoogleMapReact` components in project and you want to use heatmap layer so provide `heatmapLibrary={true}` for all `GoogleMapReact` components so component will load heatmap library at the beginning with google map api.
+If you have multiple `GoogleMapReact` components in project and you want to use heatmap layer provide `libraries:['visualization']` to `bootstrapURLKeys` so that the visualization library will be loaded with the google map api.
```

Comment:
I think we should do something like:

> If you have multiple instances of the map in your project and you are willing to use the heatmap layer in at least one of them, make sure to provide `libraries:['visualization']` to `bootstrapURLKeys` so that the visualization library will be loaded with the google map api.

Maybe we could ask @NerdCowboy or @karldanninger to help us with the wording

Reply:
Oh a native touch could be good! wording docs is always tricky.

Reply:
I've updated it with my recommendations in case they take some time to respond, I will merge it if we don't get an answer in the following days.

I will hit up @karldanninger in a PM to check if he will be available soon!

Reply:
:+1:

Reply:
Hey @itsmichaeldiego & @aviyadeveloper, thanks for doing this, looks awesome!

Some edits:

If you have multiple ~~instances of a~~ **maps** in your project and ~~you are willing to use the~~ require **a** heatmap layer in at least one of them, provide `libraries:['visualization']` to `bootstrapURLKeys` ~~so that the~~ and the visualization library will be loaded with the google map api.

Reply:
Updated! Let me know what you think @karldanninger @aviyadeveloper

Reply:
looks great @itsmichaeldiego @karldanninger :rocket:

Reply:
@aviyadeveloper Did you test everything thoroughly? I need some time to give it a test and I will merge it and release after that

Reply:
@itsmichaeldiego I would prefer you testing it than me. Since we already had the problems in the last PR and as I am new to the code-base and also pretty busy with work. When I have more time I will check out the testing mechanism and familiarize myself with that process better for future PRs to be stricter and more reliable. But this one should definitely pass your hands first.

Reply:
Of course! I will test it thoroughly this time, just wondering if you were able to check those!